node/package.json

```diff
@@ -17,7 +17,7 @@
     "test:all": "npm run test:format && npm run test:lint && npm run test:audit && npm run test:types && npm test",
     "test:audit": "npm audit --audit-level=moderate --production",
     "test:format": "prettier --check '**/*.{js,jsx,json,ts,tsx}'",
-    "test:lint": "eslint src --ext ts --ext js --max-warnings=0",
+    "test:lint": "eslint src --ext ts --ext js --max-warnings=25",
```

Comment:
exactly 25?

Reply:
I removed a bunch of config that was disabling lint rules (like 'no any'), and then I fixed most of the errors, but left 25. The idea is don't let anyone add *more* lint errors. Eventually would be nice to fix and get it to 0